### ISSUE TITLE
Training setting.

### ISSUE THREAD

**Issue by gbaizj824** [suhwan-cho/CRVOS#2]
Thanks for your great work!
I have a question about pre-training. I understand the Clue uses the predicted mask of the last frame as input to predict the mask of the current frame. However, the Youtube-VOS train set is labelled every fifth frame. So, how do you define the previous mask? Or do nothing, just use the previous adjacent mask as input?

**Comment by suhwan-cho** [comment]
Hi, thanks for your attention!
The frames only with the annotations are used for pre-training. 
For example, frame number 0, 5, 10, 15, ...  are considered as frame number 0, 1, 2, 3, ... in the pre-training stage.

**Comment by suhwan-cho** [comment]
Hi, thanks for your attention!
The frames only with the annotations are used for pre-training. For example, frame number 0, 5, 10, 15, ... are considered as frame number 0, 1, 2, 3, ... in the pre-training stage.

**Comment by gbaizj824** [comment]
Thanks,closed.
